packages/application-components/src/components/modal-pages/internals/modal-page-header.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import requiredIf from 'react-required-if';
+import {
+  customProperties,
+  Text,
+  Spacings,
+  PrimaryButton,
+  SecondaryButton,
+} from '@commercetools-frontend/ui-kit';
+import { css } from '@emotion/core';
+import filterDataAttributes from '../../../utils/filter-data-attributes';
+
+const ModalPageHeader = props => (
+  <div
+    css={css`
+      display: flex;
+      align-items: flex-end;
+      justify-content: space-between;
+      margin: 0 ${customProperties.spacingM};
+      padding: ${customProperties.spacingM} 0;
+      border-bottom: 1px solid #afafaf;
+      & > * + * {
+        margin-left: ${customProperties.spacingM};
+      }
+    `}
+  >
+    <div
+      css={css`
+        overflow: hidden;
+        & * + * {
+          margin-top: ${customProperties.spacingM} !important;
```

Comment:
This is basically for the same purpose as using a `<Spacings>` component, but I couldn't use it because I needed to have `overflow: hidden` in order to properly truncate the text of its children.